.kitchen.yml

```diff
@@ -19,3 +19,4 @@ suites:
       - recipe[sudo]
       - recipe[user::data_bag]
       - recipe[osl-app::app1]
+      - recipe[osl-app::app2]
```

Comment:
You still need to make the actual databags for each user in the repo for testing as TK fails when I try to run it.